classes/04class/exercises/c04-iac02/carloshz4/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "region" {
+  description = "Define the region for providers"
+}
+
+variable "aws_ip_cidr_range" {
+  type        = "string"
+  description = "IP CIDR range for AWS VPC"
+}
+
+variable "subnet-name" {
+  type        = "map"
+  description = "Name tags for the different subnets"
+}
+
+variable "subnet-cidr" {
+  type        = "map"
+  description = "CIDR blocks the different subnets"
+}
```

Comment:
subnet `name` and `cidr` are related, I suppose.

Having separate inputs for these might lead to problems.

You could have a single `map` with `object`.

Example:
```
map(object({
   name = string
   cidr = string
})
)
```

https://www.terraform.io/docs/configuration/variables.html#declaring-an-input-variable